src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -62,6 +62,11 @@ def _get_detail_error(self, ex):
         except: #pylint: disable=bare-except
             return ex
 
+def list_locations(sku):
+    client = web_client_factory()
+    result = client.list_geo_regions(sku)
+    return [{'name': x.name} for x in result]
```

Comment:
This can go in `cli_command(..., transform=)` instead of creating a custom command only to change the return object.